lms/djangoapps/bulk_email/tests/test_tasks.py

```diff
@@ -278,6 +278,36 @@ def test_ses_domain_ends_with_dot(self):
         # Test that celery handles permanent SMTPDataErrors by failing and not retrying.
         self._test_email_address_failures(SESDomainEndsWithDotError(554, "Email address ends with a dot"))
 
+    def test_email_address_with_ascii_characters_failed(self):
```

Comment:
Suggested name `test_bulk_email_skip_with_non_ascii_emails` seems better for this test.

Reply:
okay @noraiz-anwar i will change it.